scripts/scil_visualize_bundles_mosaic.py

```diff
@@ -0,0 +1,248 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+import argparse
+import os
+import shutil
+import nibabel as nib
+
+from scilpy.io.utils import (
+    add_overwrite_arg, assert_inputs_exist, assert_outputs_exists)
+
+from dipy.viz import actor, window
```

Comment:
Third-party imports should be before imports specific to this project (i.e. the scilpy imports). Please move them before, in a block with Nibabel